lib/atlas/initializer_input.rb

```diff
@@ -0,0 +1,23 @@
+module Atlas
+  class InitializerInput
+    include ActiveDocument
+
+    DIRECTORY = 'initializer_inputs'
+
+    attribute :query,           String
+    attribute :share_group,     Symbol
```

Comment:
If we have the `share_group` in here - and I think we should - we also need a validation (which is run in the specs of etsource) that all the inputs (of one derived dataset) for a share_group sum up to 100%. If you specify one input from a group, you have to specify all of them - because we do not have a balancing mechanism like in etengine's `ScenarioUpdater#balanced_values`